src/assets/style/ShortcutGuideStyle.css

```diff
@@ -4,101 +4,138 @@
  * You should have received a copy of the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
  * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
  */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
 
-/* The Modal (background) */
-.a11y_modal {
-	display: none; /* Hidden by default */
-	position: fixed; /* Stay in place */
-	z-index: 1000; /* Placed on top */
+.shortcut-guide__background {
```

Comment:
Let's consider the API we want to expose externally. We can improve our approach by:
1. Defining color properties as CSS variables on the top-level shortcut-guide element:

```css
.shortcut-guide {
  --fdz-shortcut-guide-background: rgba(0, 15, 58, 0.35);
  --fdz-shortcut-guide-container: #fff;
  --fdz-shortcut-guide-border-radii: 8px;
}
```
Benefits:
- Allows external developers to easily override these values
- Promotes consistency and simplifies maintenance

2. Utilizing the native <dialog> element:
The <dialog> element is now supported in all evergreen browsers and offers several advantages:
- Provides a better HTML experience
- Handles modal behavior natively
- Improves accessibility out of the box

Recommendation:
Consider implementing the shortcut guide as a native <dialog> element. This approach would leverage built-in browser capabilities and potentially simplify our code.
For more information on the <dialog> element, refer to: [web.dev/learn/html/dialog](https://web.dev/learn/html/dialog)
By adopting these changes, we can create a more flexible, maintainable, and standards-compliant implementation of the shortcut guide.